.github/workflows/documentation.yml

```diff
@@ -0,0 +1,15 @@
+name: documentation
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
```

Comment:
Is Python Semver trustful enough to live it open to any `3.x` version? Should we pin the latest, 3.8 for example?

Reply:
Good point indeed.